Gopkg.lock

```diff
@@ -795,6 +796,7 @@
     "github.com/pkg/errors",
     "github.com/prometheus/client_golang/prometheus",
     "github.com/prometheus/client_golang/prometheus/promhttp",
+    "github.com/prometheus/client_golang/prometheus/testutil",
```

Comment:
This is interesting... We have our, Thanos `testutil` package (:  Also even though, why this change if no update was needed in this PR? ): Maybe you need `rebase` with master?

Reply:
Rebased. No change.
dep ensure puts this back each and every time.

```
dep:
 version     : v0.5.0
 build date  : 2018-07-26
 git hash    : 224a564
 go version  : go1.10.3
 go compiler : gc
 platform    : linux/amd64
 features    : ImportDuringSolve=false
```

Reply:
can you use `make deps` not `dep ensure`? We use pinned dep version (:

Reply:
yes, looks like not needed, but not harmful either